static/src/js/components/Button/Button.js

```diff
@@ -77,9 +77,14 @@ export const Button = ({
     rel = 'noopener nofollow'
   }
 
+  let asEl
```

Comment:
Seems like this would always be undefined?

Reply:
The bootstrap button has some built in logic to support hrefs on buttons. This basically says, if `as` is set to `button` and an `href`, let's use bootstraps logic instead